openssl-sys/src/x509v3.rs

```diff
@@ -58,15 +58,25 @@ pub const EXFLAG_FRESHEST: u32 = 0x1000;
 #[cfg(any(ossl102, libressl261))]
 pub const EXFLAG_SS: u32 = 0x2000;
 
+#[cfg(not(boringssl))]
```

Comment:
We don't use the openssl-sys bindings with boringssl.

Reply:
Removed the bindings.